minio/helpers.py

```diff
@@ -187,6 +187,21 @@ def read_full(data, size):
 
     return chunk.getvalue()
 
+
+def read_complete_file_in_chunks(data, size):
+    """
+    read complete file in chunks
+    """
+    content = read_full(data, size)
+    read_size = len(content)
+    yield content
+
+    while read_size >= size:
```

Comment:
Can we have `while read_size == size` instead? `read_full` exactly reads `size` bytes from the reader

Reply:
That's possible. I'll change it this evening(Europe/Berlin).